src/main/java/com/slack/kaldb/logstore/LogStore.java

```diff
@@ -24,6 +24,17 @@
 
   Path getDirectory();
 
-  Collection<String> activeFiles();
+  /**
+   * After a commit, lucene may merge multiple segments into one in the background. So, getting a listing of files in a lucene index will not return a stable set of files. 
+   * To address this, we can get an IndexCommit, which provides a consistent set of files at a given point in time and also protects the files from deletion.
+   * If a background merge happens after an indexCommit is created, lucene holds on to those files on disk even if the background merge has marked that file for deletion. 
+   * So, we need to release the index commit object once it's no longer needed using the releaseIndexCommit api call. 
+   * Since lucene can always decide to run background merges on an index, multiple calls to getIndexCommit, may return different lists of active files, even if no new commits are performed on the index.
+   */
+  public IndexCommit getIndexCommit();
+
+  public void releaseIndexCommitRef(IndexCommit indexCommit);
```

Comment:
Can you please rename this method releaseIndexCommit to be consistent with the get call.

Reply:
> Let's rename the function releaseIndexCommitRef to better reflect it's intended function.

I renamed it to what you suggested last night :)

Reply:
Sorry. I thought the Ref suffix was in the get call also. Can we drop the suffix here?